src/WrappedMToken.sol

```diff
@@ -641,10 +627,13 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
             IRegistrarLike(registrar).listContains(_EARNERS_LIST, account_);
     }
 
+    /**
+     * @dev    Returns the projected total earning supply if all accrued yield was claimed at this moment.
+     * @param  currentIndex_ The current index.
+     * @return supply_       The projected total earning supply.
+     */
     function _projectedEarningSupply(uint128 currentIndex_) internal view returns (uint240 supply_) {
-        // NOTE: Round up to overestimate the earning supply plus all accrued yield, such that:
-        //       `projectedEarningSupply_ >= Sum of all (present earning balances + accrued yield)`.
-        return IndexingMath.getPresentAmountRoundedUp(_principalOfTotalEarningSupply, currentIndex_);
+        return IndexingMath.getPresentAmountRoundedDown(_principalOfTotalEarningSupply, currentIndex_);
```

Comment:
this will lead to a bigger M excess in wrapper 
what is the argument for round down?

Reply:
discussed:

`we definitely don't want our total accrued yield to be less than the sum of all accrued yield`